shared/chat/conversation/messages/system-git-push/index.js

```diff
@@ -76,11 +76,13 @@ class GitPush extends React.PureComponent<Props> {
                       <Text
                         type="Terminal"
                         selectable={true}
-                        style={{
-                          color: globalColors.blue,
-                          fontSize: 11,
-                          lineHeight: isMobile ? 16 : 1.3,
-                        }}
+                        style={platformStyles({
+                          common: {
+                            color: globalColors.blue,
+                            fontSize: 11,
+                            lineHeight: 16,
```

Comment:
meant to remove the differing lineHeight for mobile?

Reply:
so lineheight is kinda weird. on mobile is unitless (really a number) and always means pixels. on desktop we 99.9% mean px but that means we need some kind of helper to always ensure its a string with 'px' on the end, so we had lineHeight(12) calls to do this conversion. Desktop does support lineheight w/o units but then its a function of the actual font size (which we basically never ever do) so really, this 1.3 is basically trying to be 16px, i don't see why it'd be anything else